compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/MockDesugar.java

```diff
@@ -213,6 +246,7 @@ private BLangInvocation generateBLangInvocation() {
 
     private BInvokableSymbol getMockHandlerInvokableSymbol() {
         BSymbol testPkg = bLangPackage.getTestablePkg().symbol.scope.lookup(new Name("test")).symbol;
+
         BInvokableSymbol mockHandlerSymbol =
```

Comment:
`mockHandlerSymbol` local variable seems redundant

Reply:
I could compress the code into a single line, however, that would make it difficult to read. It would be something like `(BInvokableSymbol) bLangPackage.getTestablePkg().symbol.scope.lookup(new Name("test")).symbol.testPkg.scope.lookup(new Name("mockHandler")).symbol`.